meson.build

```diff
@@ -4,6 +4,7 @@ project(
 	version: '3.0.1',
 	meson_version: '>=0.54.0',
 	license: 'MIT',
+	subproject_dir: 'external',
```

Comment:
Was this necessary, in the end? Is this what makes `catch2_dep = dependency(...)` etc work?